relayer-cli/src/commands/query.rs

```diff
@@ -40,6 +40,10 @@ pub enum QueryClientCmds {
     /// The `query client connections` subcommand
     #[options(help = "query client connections")]
     Connections(client::QueryClientConnectionsCmd),
+
+    /// The `query clients` subcommand
+    #[options(help = "query client all")]
+    All(client::QueryAllClientsCmd),
```

Comment:
Could we have `query clients ..` instead of `query client all`? i.e. move the subcommand one level up.

Reply:
ok